src/module.ts

```diff
@@ -246,16 +246,26 @@ class Ctrl extends PanelCtrl {
     return datasourceRequest;
   }
 
-  async export(panelId: string, target: any): Promise<void> {
-    // TODO: support org_id
-    let panelUrl = window.location.origin + window.location.pathname + `?panelId=${panelId}&fullscreen`;
+  async export(panelId: string | null, target: any): Promise<void> {
```

Comment:
it's not very obvious that you should use `null` if you want to export data from all panels

let's add a docstring at least :)